code/modules/projectiles/guns/energy.dm

```diff
@@ -20,6 +20,12 @@
 	/// Do you want the gun to fit into a turret, defaults to true, used for if a energy gun is too strong to be in a turret, or does not make sense to be in one.
 	var/can_fit_in_turrets = TRUE
 
+/obj/item/gun/energy/examine(mob/user)
+	. = ..()
+	if(cell)
+		. += "<span class='notice'>It is [round(cell.percent())]% charged.</span>"
```

Comment:
I feel it would look nicer if "Use a pen on it to rename it" followed the charge line.
But I'm not sure if something as cursed as 

```suggestion
		(.).Insert(length(.), "<span class='notice'>It is [round(cell.percent())]% charged.</span>")
```

would be worth writing to enable that...

Reply:
yeah don't do that. this syntax is cursed. also putting the weapon charge at the bottom makes the information easier to read in a high stress situation.